### DISCUSSION TITLE
Adjust Any Meida server to work with slow internet

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6361] [General]
how to Adjust Any Meida server to work with slow internet , so if we got 0.50x broadcast speed , i dont want AMS to drop the connection , not cut the recording cause we are getting recording each 1 min , so the files recorded are 1 min long

**Comment by amarantmeida** [comment]
@brixxnatt Adaptive Bitrate Streaming, also referred to as dynamic adaptive streaming or multi-bitrate streaming, enables the delivery of optimal video quality by adjusting to the network bandwidth between the viewer and the media server. This technology ensures smooth video playback regardless of the viewer's internet connection speed or device. Please review the below guide for more details
https://antmedia.io/docs/guides/adaptive-bitrate/adaptive-bitrate-streaming/
